schema/me/conversation.js

```diff
@@ -59,6 +59,48 @@ export const MessageType = new GraphQLObjectType({
   },
 })
 
+export const AttachmentType = new GraphQLObjectType({
+  name: "AttachmentType",
+  desciption: "Fields of an attachment (currently from Radiation)",
+  fields: {
+    id: {
+      description: "Attachment id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    content_type: {
+      description: "Content type of file.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    file_name: {
+      description: "File name.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    download_url: {
+      descrpition: "URL of attachment.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+  },
+})
+
+export const MessageDetailType = new GraphQLObjectType({
+  name: "MessageDetailType",
+  description: "Complete details of a message (currently from Radiation)",
+  fields: {
+    id: {
+      description: "Radiation id.",
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    attachments: {
+      description: "Attachments.",
+      type: new GraphQLList(AttachmentType),
+    },
+    raw_text: {
```

Comment:
Yeah a `created_at` timestamp should be fine; here's what the emission spec looks like (maybe we can format the timestamp here in mp as well?) :
<img width="622" alt="screen shot 2017-06-21 at 3 53 34 pm" src="https://user-images.githubusercontent.com/2712962/27387736-d7ff5338-5699-11e7-8f42-4c91bae7b87d.png">